docs/spark/risk-estimation.md

```diff
@@ -0,0 +1,74 @@
+## Invoking the Risk Estimation Spark executor
+
+The Spark JAR was built for Java 1.8 and Spark 2.2. The toolkit is packaged together with its dependencies so there is no requirement for additional configuration of classpaths or internet connection to artifact repositories.
```

Comment:
this is not consistent with what we said in https://github.com/IBM/data-privacy-toolkit/pull/101/files#diff-a80b1b3e7affdb884c33245ed1fca2f940c94a451d90b53230d4dbf013360d2aR5 (see my previous comment)

Reply:
Requires update, missed it